vnext/Shared/Threading/BatchingQueueThread.h

```diff
@@ -50,6 +50,8 @@ struct BatchingQueueThread final : facebook::react::BatchingMessageQueueThread {
 
  private:
   std::mutex m_mutex;
+  std::mutex m_mutexQuitting;
```

Comment:
>std::mutex m_mutexQuitting; [](http://example.com/codeflow?start=2&length=27)

It is better to not add a new mutex to avoid any possible deadlock situations and use the m_mutex instead.

Reply:
Keeping it the same as the master change that I'm porting - to avoid additional risk to the backport.